src/applications/appeals/testing/pages/evidenceUpload.jsx

```diff
@@ -27,6 +29,18 @@ export const evidenceUpload = {
       },
     },
   },
+
+  review: data => ({
+    'Evidence to be reviewed by the Board': (
+      <ul>
+        {data.evidence.map(file => (
+          <li key={file.name}>
+            {file.name} ({displayFileSize(file.size)})
+          </li>
+        ))}
```

Comment:
Should this have an `else statement` for if `no files are uploaded`?

Reply:
FIxed